packages/apps/src/microsoft/teams/apps/app.py

```diff
@@ -206,22 +206,21 @@ async def start(self, port: Optional[int] = None) -> None:
                 if hasattr(plugin, "on_init") and callable(plugin.on_init):
                     await plugin.on_init()
 
-            # Start all plugins except HTTP plugin first
-            for plugin in self.plugins:
-                is_callable = hasattr(plugin, "on_start") and callable(plugin.on_start)
-                if plugin is not self.http and is_callable:
-                    event = PluginStartEvent(port=self._port)
-                    await plugin.on_start(event)
-
             # Set callback and start HTTP plugin
             async def on_http_ready() -> None:
                 self.log.info("Teams app started successfully")
                 assert self._port is not None, "Port must be set before emitting start event"
                 self._events.emit("start", StartEvent(port=self._port))
 
             self.http.on_ready_callback = on_http_ready
-            event = PluginStartEvent(port=self._port)
-            await self.http.on_start(event)
+
+            tasks: List[Awaitable[Any]] = []
```

Comment:
should this be in app.plguins?

Reply:
i think it makes more sense in app.py cause only the app class should have the power to start the plugins